docs/pom.xml

```diff
@@ -218,6 +218,11 @@
         <artifactId>junit</artifactId>
         <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
I know other parts of the ecosystem use opencsv, like for example gs-wfs:

```
    <dependency>
      <groupId>net.sf.opencsv</groupId>
      <artifactId>opencsv</artifactId>
      <version>1.8</version>
      <scope>test</scope>
    </dependency>
```
and there are some in app-schema too.

Well it's a test and it's unlikely to cause conflicts (only importer depends on gt-csv right?), but something to keep in mind and maybe verify.

Reply:
And oh, the switch will require a corresponding PR in GeoServer to upgrade the gt-csv dependencies, in particular, the package.

Reply:
I think GeoServer will be OK for the time being - https://osgeo-org.atlassian.net/browse/GEOS-9439